### PR TITLE
Gdr 1924

### PR DESCRIPTION
**Pull request by darsoo**
# Description
## What changed?
Related JIRA issue: 

## Why was it changed?

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [x] Package version bumped
- [x] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]
GJ!

File: R/pipeline_helpers.R [hunk 1]
```diff
@@ -175,3 +175,65 @@ paste_warnings <- function(list, sep = "\n") {
...
+  df_2 <- df_2[df_2$DrugName != df_2$DrugName_2, ]
+  rows <- df_2$Concentration_2 > 0
+  cols <- c("Concentration", "Concentration_2")
+  df_2[rows, cols] <- df_2[rows, cols] / 2
+  
+  df_2
+}
+
+#' @keywords internal
+changeColNames <- function(df, drugs, suffix) {
```
Comment (bczech) [inline R/pipeline_helpers.R:225]: @darsoo where this function is used?

File: R/pipeline_helpers.R [hunk 2]
```diff
@@ -175,3 +175,65 @@ paste_warnings <- function(list, sep = "\n") {
...
+  df_layout_2 <- merge(df_layout, df_2, by = NULL)
+  if (modifyDf2) {
+    df_layout_2 <- df_layout_2[!(df_layout_2$Concentration == 0 & df_layout_2$Concentration_2 > 0), ]
+  }
+  
+  gDRtestData::generate_response_data(df_layout_2, noise)
+}
+
+#' @keywords internal
+prepareCodilutionData <- function(df, df_layout) {
```
Comment (bczech) [inline R/pipeline_helpers.R:213]: @darsoo where this function is used?

File: R/pipeline_helpers.R (hunk 1 above)
Comment (bczech) [reply R/pipeline_helpers.R:225]: Ok, `generate_wrappers.R` was hidden in the PR :) Nevermind

File: R/pipeline_helpers.R (hunk 2 above)
Comment (bczech) [reply R/pipeline_helpers.R:213]: Ok, `generate_wrappers.R` was hidden in the PR :) Nevermind

**Pull request merged**